ci-operator/step-registry/ipi/install/workers-rhel7/scaleup/ipi-install-workers-rhel7-scaleup-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+# Ensure our UID, which is randomly generated, is in /etc/passwd. This is required
+# to be able to SSH.
+if ! whoami &> /dev/null; then
+    if [[ -w /etc/passwd ]]; then
+        echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+    else
+        echo "/etc/passwd is not writeable, and user matching this uid is not found."
+        exit 1
+    fi
+fi
+
+# Dump the Ansible inventory for debugging
+ansible-inventory -i "${SHARED_DIR}/ansible-hosts" --list --yaml
+
+# Run the scaleup playbook
+ansible-playbook -i "${SHARED_DIR}/ansible-hosts" playbooks/scaleup.yml -vvv
+
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+# Drain CoreOS nodes
+mapfile -t COREOS_WORKERS < <(oc get nodes -o wide | grep worker | grep "Red Hat Enterprise Linux CoreOS" | awk '{print $1}' || true)
+if [[ ${#COREOS_WORKERS[@]} != 0 ]]
+then
+    oc adm drain "${COREOS_WORKERS[@]}" --force --delete-local-data --ignore-daemonsets --timeout=0s
+fi
```

Comment:
Are we draining because the drain triggered by deleting machines is too slow??

Reply:
I was doing the drain first just because that has been my process in practice.  I can drop the drain and just go straight to the delete if that should be just as effective.

Reply:
I dropped the drain and the delete worked just fine.